p/themes/Origine/origine.css

```diff
@@ -1186,6 +1186,7 @@ a.btn {
 		padding: 3px 5px;
 		max-width: 97%;
 		width: 90px;
+		line-height: 25px;
```

Comment:
That should preferably probably be something like 1.5 or 1.6, but depending on intent it might be better to instease the top/bottom padding instead? (`padding: 16px 5px` for example)

Reply:
fixed